apps/edinburghbus/edinburgh_bus.star

```diff
@@ -0,0 +1,139 @@
+"""
+Applet: Edinburgh Bus
+Summary: Shows the next 3 buses
+Description: Give it an Edinburgh stop ID and it will show the next 3 arrivals for the next 3 services at that stop.
```

Comment:
Based on the description, I'm guessing there should be a `get_schema()` in here somewhere? ([docs](https://tidbyt.dev/docs/reference/schema))

Reply:
Of course, good call!  Will update/

Reply:
Added a config in latest commits - will look at fixing the email issue next